a9.cpp

```
@@ -0,0 +1,8 @@
+#include<bits/stdc++.h>
+using namespace std;
+int main(){
+    int div=100;
+    int ans1=div/0;
```

cppcheck (2.10-2)

[error] Division by zero.

(zerodiv)